pandas/_libs/tslib.pyx

```diff
@@ -87,11 +87,6 @@ def _test_parse_iso8601(ts: str):
 
     obj = _TSObject()
 
-    if ts == "now":
-        return Timestamp.utcnow()
-    elif ts == "today":
-        return Timestamp.now().normalize()
-
```

Comment:
this function is only used for testing of the ISO fastpath. there's no test that has `'now'` or `'today'`, and they wouldn't test the ISO fastpath anyway, so might as well remove them